ActindoWorkFlow/Operation/01_ManageWorkflows.md

```diff
@@ -38,9 +42,9 @@ No prerequisites to fulfill.
 
 4. Click the *Choose the data type of your start place* field and enter the name of the place or a keyword you are searching for.
 
-    > [Info] The list of places is filtered for your keyword as you type.
+    > [Info] The list of places is filtered for your keyword as you type. To decide on the data type, see the data models at *Dev Tools > API > Tab Data Model* and find the data model that matches your needs. For example, if you want to process business document types, check the data models in the *Fakturierung* section and search for **BusinessDocument**.
```

Comment:
Tab DATA MODELS (groß geschrieben)
Ich weiß nicht, ob das hilft oder verwirrt. In Data Models heißt es leider nicht Fakturierung, sonders RETAILSUITEFAKTBASE (s. Screenshot). Wenn wir diese Info geben, vielleicht den Namen auf Deutsch geben und in Klammern (Order Management)?  Unglücklicherweise sieht man auch nicht den ganzen Namen von den Business Documents in Data Models, daher weiß ich nicht, ob diese Info wirklich hilfreich ist.

![image](https://github.com/Actindo-AG/Documentation/assets/99268392/ebe55415-4759-4ac3-baa6-5857dcd54e43)